### DISCUSSION TITLE
Laravel 9 Mailable Markdown Render issue

### DISCUSSION THREAD

**Discussion by discoverlance-com** [laravel/framework#45336] [Q&A] [answered]
I am using laravel 9 and having an issue with the mailable markdown. I created a mail with a markdown content:

```blade
<x-mail::message>
    # Hello {{ $name }},

    Please click on the link below to login

    <x-mail::panel>
        Temporay Password: {{ $password }}
    </x-mail::panel>

    <x-mail::button :url="$register_url">
        Login
    </x-mail::button>

    Thanks,<br>
    {{ config('app.name') }}
</x-mail::message>

```

Unfortunately, after testing the mail, I get the following:

![image](https://user-images.githubusercontent.com/52635726/207982989-31c80d4a-b7e3-465e-aa82-526c9324f1f8.png)

**Comment by discoverlance-com** [comment, marked as answer]
I just realized that apparently, when the mail markdown is formatted, it gives spaces to the markdown text and components, that's when I get that error. Perhaps we should take a look at this once more as most IDEs will just format the mail so it's worth perhaps noting this in the laravel docs or making a change to reflect the IDE formatting affecting the mail markdown content.

This is what I ended up with form my mail markdown after removing the spaces:

```blade
<x-mail::message>
# Hello {{ $name }},

Please click on the link below to login

<x-mail::panel>
Temporay Password: {{ $password }}
</x-mail::panel>

<x-mail::button :url="$register_url">
Login
</x-mail::button>

Thanks,<br>
{{ config('app.name') }}
</x-mail::message>
```

**Reply by milosb793** [reply]
OMG thank you! This is very sad tho

**Reply by Noyoule** [reply]
Thank you !!!
